packages/scope-sdk/tests/sdk.test.ts

```diff
@@ -1,19 +1,27 @@
 import { clusterApiUrl, Connection } from '@solana/web3.js';
-import { Scope } from '../src';
+import { Scope, scopeTokenToMint, SupportedTokens } from '../src';
+import { expect } from 'chai';
 
 describe('Scope SDK Tests', () => {
-  let connection: Connection;
-  const cluster = 'mainnet-beta';
-
-  beforeAll(() => {
-    connection = new Connection(clusterApiUrl(cluster));
-  });
+  // let connection: Connection;
```

Comment:
do you need this or can be removed?

Reply:
Not really needed for unit tests, will remove